src/rules/selector-type-case/__tests__/index.js

```diff
@@ -0,0 +1,76 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("lower", tr => {
+    
+  tr.ok("a { display: inline-block; }")
+  tr.ok("div { display: inline-block; }")
+  tr.ok("body { display: inline-block; }")
```

Comment:
Can you probably remove these redundant tests. Just one or two lowercase simple type selectors would suffice.